uds/database/data_record/mapping_data_record.py

```
@@ -1,72 +1,103 @@
+
+    Inherits from RawDataRecord to provide fallback behavior when no mapping is defined.
+    """
 
-        :param name: Name to assign to this Data Record.
-        :param length: Number of bits that this Text Table Data Record is stored over.
-        :param mapping: Bidirectional translation between raw value (int) and meaningful value (e.g. float, str).
+    def __init__(self,
+                 name: str,
+                 length: int,
+                 values_mapping: Dict[int, str],
+                 children: Sequence[AbstractDataRecord] = tuple(),
```

Ruff (0.12.2)

Unnecessary `tuple()` call (rewrite as a literal)

Rewrite as a literal

(C408)